app/assets/javascripts/ideas.js

```diff
@@ -96,6 +97,11 @@ var Ideas = {
             .success(function() { Ideas.loadAll(); })
     },
 
+    search: function(event) {
```

Comment:
I moved this higher to visually group features together and move the helper to the bottom, where it belongs.